apps/DisplayCluster/MultiTouchListener.cpp

```diff
@@ -114,6 +102,14 @@ void MultiTouchListener::refresh( TUIO::TuioTime )
 
 QPointF MultiTouchListener::_getScreenPos( TUIO::TuioCursor* tcur ) const
 {
+    return QPointF( tcur->getX() * _wallSize.width() * _targetWindow->getWallScale() +
+                    _targetWindow->getWallOffset().x() + _targetWindow->getMullionWidth(),
+                    tcur->getY() * _wallSize.height( ) * _targetWindow->getWallScale() +
+                    _targetWindow->getWallOffset().y() + _targetWindow->getMullionWidth());
```

Comment:
Should be MulliionHeight here in fact, but I don't think Mullion compensation is needed at all.
Instead of all these getters, I would move the logic to a function in DisplayGroupView which converts from wallPos to screenPos.